### PR TITLE
Typos and grammar

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
Typos and grammar for managing repos topic.
## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

** Ping a doc team member to review pull request (`@joel-hamill`, `@emanic`, or `@sascala`) **

### CODE REVIEW

File: 1.7/usage/repo.md [hunk 1]
```diff
@@ -6,18 +6,18 @@ menu_order: 50
...
+* [Search for packages](#finding-packages)
+* [Add repositories](#adding)
+* [Remove repositories](#removing)
 
 **Tip:** Before you can use the CLI, you need to [install it](/docs/1.7/usage/cli/install/).
 
 ## <a name="listing"></a>Listing repositories
 
-To see which package repositories are currently configured from the DC/OS CLI as follows:
+To see which package repositories are currently configured, enter this CLI command:
```
Comment (sascala) [inline 1.7/usage/repo.md]: Not to be picky, but maybe s/configured/installed ?

File: 1.8/usage/repo.md [hunk 2]
```diff
@@ -6,18 +6,18 @@ menu_order: 50
...
+* [Search for packages](#finding-packages)
+* [Add repositories](#adding)
+* [Remove repositories](#removing)
 
 **Tip:** Before you can use the CLI, you need to [install it](/docs/1.8/usage/cli/install/).
 
 ## <a name="listing"></a>Listing repositories
 
-To see which package repositories are currently configured from the DC/OS CLI as follows:
+To see which package repositories are currently configured, enter this CLI command:
```
Comment (sascala) [inline 1.8/usage/repo.md]: same as above

File: 1.9/usage/repo.md [hunk 3]
```diff
@@ -6,18 +6,18 @@ menu_order: 50
...
+* [Search for packages](#finding-packages)
+* [Add repositories](#adding)
+* [Remove repositories](#removing)
 
 **Tip:** Before you can use the CLI, you need to [install it](/docs/1.9/usage/cli/install/).
 
 ## <a name="listing"></a>Listing repositories
 
-To see which package repositories are currently configured from the DC/OS CLI as follows:
+To see which package repositories are currently configured, enter this CLI command:
```
Comment (sascala) [inline 1.9/usage/repo.md]: same comment

**Review by sascala — COMMENTED** [review]
1 small comment 3 times

**Comment by joel-hamill** [comment]
@sascala done

**Pull request merged**